src/vm/comutilnative.cpp

```diff
@@ -712,6 +712,22 @@ FCIMPL6(void, GCInterface::GetMemoryInfo, UINT64* highMemLoadThreshold, UINT64*
 }
 FCIMPLEND
 
+void QCALLTYPE GCInterface::GetGCConfigValue(const wchar_t* key, QCall::StringHandleOnStack result)
```

Comment:
@jkotas Is this the best way to return a string from a function? `GCHeap::GetGCConfigValue` has to always return a newly allocated string, which is then put in to a string holder here, `result.Set` is called, then it's deleted. (If we don't always return a newly allocated one, how would we know whether it should be deleted or not?) Alternately we could try to pass the StringHandleOnStack down to gc.cpp, but I don't think it has the necessary headers to deal with those.

Reply:
> GCHeap::GetGCConfigValue has to always return a newly allocated string, which is then put in to a string holder here, result.Set is called, then it's deleted

Yes, that's the way to do it.